### PR TITLE
Fixes #33295 - web UI ostree sync

### PR DESCRIPTION
**Pull request by jjeffers**
This PR is support for web-ui based synchronization of OSTree repositories.

image

### CODE REVIEW

File: lib/katello/repository_types/ostree.rb [hunk 1]
```diff
@@ -18,8 +19,31 @@
...
+  generic_content_type 'ostree_summary',
+                       model_class: Katello::GenericContentUnit,
+                       pulp3_api: PulpOstreeClient::ContentSummariesApi,
+                       pulp3_model: PulpOstreeClient::ContentSummary,
+                       pulp3_service_class: Katello::Pulp3::GenericContentUnit
+
+  generic_content_type 'ostree_ref',
+                       model_class: Katello::GenericContentUnit,
+                       pulp3_api: PulpOstreeClient::ContentRefsApi,
+                       pulp3_service_class: Katello::Pulp3::GenericContentUnit
```
Comment (jlsherrill) [inline lib/katello/repository_types/ostree.rb:46]: I'm not 100% sure we need all of these types indexed.  Maybe just ostree_ref?

Comment (jlsherrill) [reply lib/katello/repository_types/ostree.rb:46]: also ostree_ref is in here 2x

**Comment by jturel** [comment]
This PR is showing up on https://projects.theforeman.org/issues/33294 ;)

**Comment by jjeffers** [comment]
> This PR is showing up on https://projects.theforeman.org/issues/33294 ;)

Cleaned that up!

**Comment by jjeffers** [comment]
[test katello]

**Pull request merged**